gui/src/components/udx/cortx-select-create-bucket.vue

```diff
@@ -163,7 +163,10 @@ export default class CortxSelectCreateBucket extends Vue {
       this.bucketList = res.data.buckets.map((bucket: any) => {
         return { label: bucket.name, value: bucket.name, s3URI: bucket.s3_uri };
       });
-    }
+    };
+    const schemas = { schemas: "s3" }
```

Comment:
Missing semicolon. Please make sure there are no lint errors before making a commit.

Reply:
This is fixed